app/Console/Commands/Environment/AppSettingsCommand.php

```diff
@@ -136,6 +137,10 @@ public function handle()
             array_key_exists($selected, self::ALLOWED_QUEUE_DRIVERS) ? $selected : null
         );
 
+        if (empty($this->config->get('pterodactyl.cloud.uuid'))) {
+            $this->variables['PTERODACTYL_CLOUD_UUID'] = (string) Uuid::uuid4();
```

Comment:
I think this could be renamed better to be `PTERODACTYL_INSTANCE_UUID` or something.

I would prefer `Uuid::uuid4()->toString()` over the `(string)` cast as well, just for consistency with the rest of the application.

Reply:
The reason it's called PTERODACTYL_CLOUD_UUID is because it's going to send this UUID to the service when it's downloading/uploading eggs, etc... so it's an ID specific to the cloud service (since we killed the generic UUID already and don't have a use for it in other places). Happy to change it though if you think we should have it be a generic UUID.

Reply:
I think we should make it more generic if possible. I don't know where else it would be used currently, but wouldn't hurt to make it usable down the road if we add more things.